styles/Datadog/abbreviations.yml

```
@@ -1,11 +1,42 @@
 extends: substitution
-message: "Use '%s' instead of abbreviations like '%s'."
+message: "Use abbreviations for common industry terms upon first mention: '%s' instead of '%s'."
```

```suggestion
message: "Use abbreviations for common industry terms: '%s' instead of '%s'."
```
I thought this might imply that you need to spell out the abbreviation _after_ the first mention, which might confuse people. I think it's okay to make it a blanket rule?